### ISSUE TITLE
'Warning' exit status is 3 instead of 2

### ISSUE THREAD

**Issue by sjourdan** [terraform-linters/tflint#528]
Hi,

According to [the README](https://github.com/terraform-linters/tflint#exit-statuses), my understanding is that a warning from the linter should return the exit status 3 (error is 2). 

Consider the following terraform code that should issue an error (bad instance type), with a return code of 2:

```hcl
variable "instance_type" {
  description = "ec2 instance type"
  default     = "m10.small"
}
```

Using tflint 0.13.1 (also 0.12), error code should be 2 and not 3: 

```shell
$ tflint
1 issue(s) found:

Error: instance_type is not a valid value (aws_instance_invalid_type)

  on ec2.tf line 3:
   3:   instance_type = var.instance_type

$ echo $?
3
```

Warnings do work correctly, though!
 
## Version

TFLint v0.13.1

**Comment by wata727** [comment]
The "Error" means a runtime error (loading, analyzing, etc). The example is an error level issue, not an error 😅

**Comment by sjourdan** [comment]
thank you for the clarification! It makes sense from a software point of view.
are you interested in implementing return codes that match the actual result of the linter? That would be super helpful for tflint integration in CI pipelines (like many linters out there).

I mean exit codes: 
- `0` could be OK, 
- `1` could be a linter error and 
- `2` a linter warning.

**Comment by wata727** [comment]
Yes, I originally intended to implement the behavior. However, at one day, I realized that the actual result was different from what I mean. See https://github.com/terraform-linters/tflint/pull/414

At this time, I decided to keep this mistake for compatibility. However, if this is inconvenient for many people, we can change the meaning of exit status at some point.

**Comment by mataneine** [comment]
@wata727 any chance to change it?

**Comment by wata727** [comment]
Opened #1191

I'm neutral about this change, but I agree that this change makes sense.
